packages/@sfx/sayt-driver-plugin/test/unit/common/sayt-driver-plugin.test.ts

```diff
@@ -16,6 +19,22 @@ describe('Sayt Driver Plugin', () => {
       autocomplete: (query, config, callback) => null,
     };
     driver = new SaytDriverPlugin();
+    validConfig = {
+      config: {
+        collection: 'backup'
+      }
+    };
+    fetchEvent = {
+      detail: {
+        query: 'shirt',
+      }
+    };
+    saytDataPayload = {
+      detail: {
+        ...fetchEvent.detail,
```

Comment:
You should consider it a red flag when you have to drill down through an object that you've create to get a value because it's one less level of control that you have. Generally, that means that the value you want should instead be put into its own variable. That applies to the rest of the tests.